acl/src/main/java/org/apache/rocketmq/acl/plain/PlainPermissionManager.java

```diff
@@ -50,9 +50,9 @@
 
     private String fileName = System.getProperty("rocketmq.acl.plain.file", DEFAULT_PLAIN_ACL_FILE);
 
-    private  Map<String/** AccessKey **/, PlainAccessResource> plainAccessResourceMap = new HashMap<>();
+    private Map<String/** AccessKey **/, PlainAccessResource> plainAccessResourceMap = new HashMap<>();
```

Comment:
Do you check your style correct, just like here.?https://github.com/apache/rocketmq/pull/2899

Reply:
> Do you check your style correct, just like here.?#2899

yes, i checked